.gitignore

```diff
@@ -13,6 +13,10 @@ purge/admin_key.json
 purge/venv
 config/synapse/synapse.yaml
 config/synapse/admin_user_cred.json
+/config/user_activity.json
```

Comment:
The slash here is totally right. It should probably also be used for the other entries (both because it's more specific and more consistent). That does not really belong into this PR, though.